tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2082,6 +1818,43 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 		);
 	}
 
+	/**
+	 * @Then /^the JSON data of the response should (not|)\s?contain (?:display|group) name "([^"]*)"(?:| and match)$/
```

Comment:
Will this work? trying to make regex look less scary :smile: 
```suggestion
	 * @Then /^the JSON data of the response should (not )?contain (?:display|group) name "([^"]*)"(?: and match)?$/
```

Reply:
i tried but did not work.